tests/lib/nested.sh

```diff
@@ -1343,6 +1343,121 @@ nested_start_core_vm_unit() {
     fi
 }
 
+
+nested_start_lxd_vm() {
+    local QEMU CURRENT_IMAGE
+    CURRENT_IMAGE=$1
+
+    snap install lxd
+    lxd init --auto
+
+    wget -q https://github.com/canonical/lxd/releases/latest/download/bin.linux.lxd-migrate.x86_64
+    chmod u+x bin.linux.lxd-migrate.x86_64
+    apt install -y rsync expect
+
+    # TODO: ask lxd team to suport configuration file for lxd-migrate tool
+    cat <<EOF > lxd-migrate-configurator.exp
+set timeout 120
+spawn ./bin.linux.lxd-migrate.x86_64
+
+expect {
```

Comment:
Although not merged yet, if you do need to continue using lxd-migrate, then you're likely going to find using the flags easier once this PR is merged and released:

https://github.com/canonical/lxd/pull/14041

Reply:
@tomponline Do you know when is going to be released?

Reply:
sadly, I am forced to use expect as secure boot option cannot be configured, and by default it is not set.

Reply:
I believe --config security.secureboot will control it.

Reply:
You can download the new binary from the edge test runs om github and itll be in lxd 6.2 release in a few weeks hopefully.